added new unit test endpoints

AlexanderWangY opened the pull request:
Changed names and lat lon positions

Added:
- broad coordinates endpoint to the second decimal point

h1divp commented:
I'm a little worried about having a getMessagesByCoordinates function alongside getMessagesBySpecificCoordinates and getMessagesByBroadCoordinates, as this looks confusing. Personally I believe we should remove getMessagesbyCoordinates since we will only be using the two other methods.

getMessagesByBroadCoordinates is written well. For testing, this would allow us to get all messages with the matching coordinate and will also be important for finding potential nearby users close to a client.

For getMessagesSpecificCoordinates, this just needs to query Firestore for a message holding the matching specific coordinates. Currently, it returns a string that includes the relative coordinate boundaries to a client, which can be better implemented in a "getNearbyUsers" function. So either change this to return a message or I would remove it.

Please make these edits, and when you're done I'll accept the PR

AlexanderWangY commented:
Finished!

- Made a new actions for getUsers
- Changed endpoints so that get requests have the route /get and post has /create (For increased clarity for maintainers)
- Deleted getMessagebyCoordinates (redundant)
- Added post and get error handling on bottom of index.ts

h1divp reviewed (approved):
Much better
